snowflakes/snowflakes.ass

```diff
@@ -14,125 +14,154 @@ Style: CHN,MonyharOS Sans SC Medium,41,&H00FFFFFF,&H000000FF,&H00000000,&H000000
+
 Dialogue: 0,0:00:09.90,0:00:11.34,English,,0,0,0,,The tips of those needles are
-Dialogue: 0,0:00:09.90,0:00:11.34,Chinese,,0,0,0,,
+Dialogue: 0,0:00:09.90,0:00:11.34,Chinese,,0,0,0,,那些针尖的直径
 Dialogue: 0,0:00:11.34,0:00:13.53,English,,0,0,0,,like a hundred nanometers in diameter.
-Dialogue: 0,0:00:11.34,0:00:13.53,Chinese,,0,0,0,,
+Dialogue: 0,0:00:11.34,0:00:13.53,Chinese,,0,0,0,,大概有100纳米
+
 Dialogue: 0,0:00:13.53,0:00:15.76,English,,0,0,0,,- [Derek] That is so wild.
-Dialogue: 0,0:00:13.53,0:00:15.76,Chinese,,0,0,0,,
+Dialogue: 0,0:00:13.53,0:00:15.76,Chinese,,0,0,0,,-「Derek」那太疯狂了
 Dialogue: 0,0:00:15.76,0:00:19.00,English,,0,0,0,,Dr. Ken Libbrecht is the snowflake guy.
-Dialogue: 0,0:00:15.76,0:00:19.00,Chinese,,0,0,0,,
+Dialogue: 0,0:00:15.76,0:00:19.00,Chinese,,0,0,0,,Ken Libbrecht 博士是一位雪花狂人
+
 Dialogue: 0,0:00:19.00,0:00:21.93,English,,0,0,0,,- I was the snowflake consultant for the movie, "Frozen."
-Dialogue: 0,0:00:19.00,0:00:21.93,Chinese,,0,0,0,,
+Dialogue: 0,0:00:19.00,0:00:21.93,Chinese,,0,0,0,,- 我曾经担任电影「冰雪奇缘」的雪花顾问
+
 Dialogue: 0,0:00:21.93,0:00:25.55,English,,0,0,0,,It's okay to conjure snowflakes out of your fingertips,
-Dialogue: 0,0:00:21.93,0:00:25.55,Chinese,,0,0,0,,
+Dialogue: 0,0:00:21.93,0:00:25.55,Chinese,,0,0,0,,生成雪花的指尖魔法是存在的
 Dialogue: 0,0:00:25.55,0:00:27.42,English,,0,0,0,,but they have to be real snowflakes,
-Dialogue: 0,0:00:25.55,0:00:27.42,Chinese,,0,0,0,,
+Dialogue: 0,0:00:25.55,0:00:27.42,Chinese,,0,0,0,,但也得是真实的雪花
+
 Dialogue: 0,0:00:27.42,0:00:28.85,English,,0,0,0,,or people aren't buying it.
-Dialogue: 0,0:00:27.42,0:00:28.85,Chinese,,0,0,0,,
+Dialogue: 0,0:00:27.42,0:00:28.85,Chinese,,0,0,0,,不然人们可不会买账
+
 Dialogue: 0,0:00:30.37,0:00:34.47,English,,0,0,0,,The US Post Office made snowflake stamps using my pictures.
-Dialogue: 0,0:00:30.37,0:00:34.47,Chinese,,0,0,0,,
+Dialogue: 0,0:00:30.37,0:00:34.47,Chinese,,0,0,0,,美国邮政局用我的雪花印了雪花邮票
 Dialogue: 0,0:00:34.47,0:00:36.39,English,,0,0,0,,It's not the kind of thing you normally think of
-Dialogue: 0,0:00:34.47,0:00:36.39,Chinese,,0,0,0,,
+Dialogue: 0,0:00:34.47,0:00:36.39,Chinese,,0,0,0,,正常人谁能想到呢
+
 Dialogue: 0,0:00:36.39,0:00:37.96,English,,0,0,0,,when you start doing physics,
-Dialogue: 0,0:00:36.39,0:00:37.96,Chinese,,0,0,0,,
+Dialogue: 0,0:00:36.39,0:00:37.96,Chinese,,0,0,0,,从事物理行业
 Dialogue: 0,0:00:37.96,0:00:39.67,English,,0,0,0,,that you'd be on a postage stamp.
-Dialogue: 0,0:00:37.96,0:00:39.67,Chinese,,0,0,0,,
+Dialogue: 0,0:00:37.96,0:00:39.67,Chinese,,0,0,0,,最后竟会和邮票扯上关系
 Dialogue: 0,0:00:39.67,0:00:42.36,English,,0,0,0,,- [Derek] You've written the book on snowflakes, literally.
-Dialogue: 0,0:00:39.67,0:00:42.36,Chinese,,0,0,0,,
+Dialogue: 0,0:00:39.67,0:00:42.36,Chinese,,0,0,0,,-「Derek」你写过一本关于雪花的书
 Dialogue: 0,0:00:42.36,0:00:45.04,English,,0,0,0,,- So I had like two successful books in a row,
-Dialogue: 0,0:00:42.36,0:00:45.04,Chinese,,0,0,0,,
+Dialogue: 0,0:00:42.36,0:00:45.04,Chinese,,0,0,0,,- 我曾经连续写出两本成功的书

```

Comment:
@ksyx I will apply your suggestion here.
The original one seemed outdated, sorry.

```suggestion
Dialogue: 0,0:00:42.36,0:00:45.04,Chinese,,0,0,0,,- 我曾连续写出两本热销书

```